tox.ini

```diff
@@ -76,29 +76,30 @@ commands =
 commands =
     python -m pytest -v --csv={toxworkdir}/results_val_{envname}.csv -m {posargs:new}
 
-
-[testenv:stability-{lin,win}]
-deps =
+[testenv:fuzzing]
+deps=
     {[testenv]deps}
-    pytest-loop
+    atheris
 allowlist_externals =
     bash
 commands_pre =
     bash -c 'cargo -V; echo "cargo (rust) version checking exit code = $?"'
-    python -m pip uninstall pytest-stress -y
 commands =
-    python -m pytest -v --csv={toxworkdir}/results-{envname}.csv -x {posargs:--loop 5}
-
+    ; coverage erase
+    python tests/fuzzing/cli_fuzzing.py {posargs:-dict=tests/fuzzing/assets/cli_operations.dict -artifact_prefix={toxworkdir}/ -print_final_stats=1 -atheris_runs=500000}
```

Comment:
what is the difference between fuzzing and stability tests?

Reply:
Stability tests repeatedly runs all tests 5 times sequentially and the results were used for the stress test report which is a requirement to the release but it was replaced with the fuzzing after we implemented it.
Fuzzing or fuzz testing is the technic to verify stability or robustness of interfaces such as CLI or API that exposed from the project and it works by passing randomly generated data to the CLI or API as the arguments.
The expectation of this changes would be that datumaro CLI provides proper help messages or usage without having any uncoughted exceptions on the different environments that setup by using different extra options while installing datumaro.

Reply:
Is there any background of having `atheris`?

Reply:
Previously, I meant that fuzzing requires 500K runs, while stability runs the same test code only for 5K times.
Is stability for testing extra dependencies? Could you let me know the test schedule for them (daily, weekly, manual, etc.)?

Reply:
fuzzing will be performed once per a release (not scheduled, on demand base) but I want to verify basic stability of the datumaro CLI on all available installation environments of datumaro those can be created through the different extra options. To do that, i don't think that needs to attempt too much trials and it's good to run with weekly schedule.